aspnetcore/blazor/get-started.md

```diff
@@ -28,19 +28,20 @@ Get started with Blazor:
 
    1\. Install the latest [Visual Studio preview](https://visualstudio.com/vs/preview) with the **ASP.NET and web development** workload.
 
-   2\. Install the latest [Blazor extension](https://go.microsoft.com/fwlink/?linkid=870389) from the Visual Studio Marketplace. This step makes Blazor templates available to Visual Studio.
+   2\. Create a new project.
 
-   3\. Create a new project.
+   3\. Select **ASP.NET Core Web Application**. Select **Next**.
 
-   4\. Select **ASP.NET Core Web Application**. Select **Next**.
+   4\. Provide a project name in the **Project name** field or accept the default project name. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
 
-   5\. Provide a project name in the **Project name** field or accept the default project name. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
+   5\. In the **Create a new ASP.NET Core Web Application** dialog, confirm that **.NET Core** and **ASP.NET Core 3.0** are selected.
```

Comment:
This step is no longer required if you select the Blazor template from the top level new project dialog.